File: app/src/main/java/com/navtej/doordashlite/di/MainActivityModule.java
package com.navtej.doordashlite.di;

import com.navtej.doordashlite.view.ui.MainActivity;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public abstract class MainActivityModule {
    @ContributesAndroidInjector
    abstract MainActivity contributeMainActivity();
}

File: app/src/main/java/com/navtej/doordashlite/di/MainActivityModule.kt
package com.navtej.doordashlite.di

import com.navtej.doordashlite.view.ui.MainActivity
import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
abstract class MainActivityModule {

    @ContributesAndroidInjector
    abstract fun contributeMainActivity(): MainActivity
}
